recipe/meta.yaml

```diff
@@ -66,7 +66,7 @@ requirements:
     - nasm  # [not (osx and x86_64)]
     # osx_64 needs yasm due to https://stackoverflow.com/questions/58796267/building-for-macos-but-linking-in-object-file-built-for-free-standing
     - yasm  # [osx and x86_64]
-    - lame  # [not win]
+    - lame
```

Comment:
```suggestion
```
If this build succeeds, i want to try to remove this entirely from the build section